### PR TITLE
bench: prepare micro benchmark to add new tests

### PR DESCRIPTION
**Pull request by ldorau**
It is only a refactor of the code.

---

This change is [Reviewable](https://reviewable.io/reviews/pmem/vmemcache/43)

### CODE REVIEW

**Comment by ldorau** [comment]
@kilobyte please review

**Comment by ldorau** [comment]
> So this is effectively a refactor?

Yes

File: benchmarks/bench_micro.c [hunk 1]
```diff
@@ -52,25 +52,63 @@ struct buffers {
...
+	for (unsigned i = 0; i < n_threads; ++i) {
+		ctx[i].cache = cache;
+		ctx[i].secs = 0.0;
+	}
+
+	return cache;
+}
+
+/*
+ * bench_fini -- (internal) initialize benchmark
```
[the remark this answers is not in the document]
Comment (ldorau) [reply benchmarks/bench_micro.c]: Done

File: benchmarks/bench_micro.c [hunk 2]
```diff
@@ -101,18 +208,18 @@ main(int argc, char *argv[])
...
 	unsigned n_threads = 10;
-	unsigned write_count = 100000;
+	unsigned ops_count = 100000;
 	size_t cache_max_size = VMEMCACHE_MIN_POOL;
 	size_t cache_fragment_size = VMEMCACHE_MIN_FRAG;
 	unsigned nbuffs = 10;
-	size_t min_size = 8;
-	size_t max_size = 64;
+	size_t min_size = 128;
+	size_t max_size = 256;
```
Comment (marcinslusarz) [inline benchmarks/bench_micro.c]: This doesn't look like "only a refactor". Such changes needs to be in a separate commit.

Comment (ldorau) [reply benchmarks/bench_micro.c]: Removed
